lib/glimesh/channel_lookups.ex

```diff
@@ -202,4 +258,38 @@ defmodule Glimesh.ChannelLookups do
   def get_any_channel_for_user(user) do
     Repo.one(from c in Channel, where: c.user_id == ^user.id)
   end
+
```

Comment:
The following method is used by the add hosts text input box type-ahead code.  Since it takes raw user input, I attempt to sanitize that input against '\\', '_', and '%' characters.